app/scripts/modules/core/src/domain/IInstance.ts

```diff
@@ -10,6 +10,7 @@ export interface IInstance {
   id: string;
   launchTime: number;
   loadBalancers?: string[];
+  name?: string;
```

Comment:
what's your opinion on having `name` be non-nullable and just mirroring the value of `id` if clouddriver doesn't have a distinct human readable name to offer?

Reply:
Ah, good point.  I've updated the IInstance interface to have a non-null `name`, and I've updated clouddriver to make sure both Load Balancer instances and Server Group instances return ID as name if a human-readable one isn't provided.